Makefile

```diff
@@ -26,7 +26,14 @@ push_images: $(addprefix push_image_, $(IMAGES))
 build_image_prognostic_run:
 	tools/docker_build_cached.sh us.gcr.io/vcm-ml/prognostic_run:$(CACHE_TAG) \
 		-f docker/prognostic_run/Dockerfile -t $(REGISTRY)/prognostic_run:$(VERSION) \
-		--target prognostic-run .
+		--target prognostic-run \
+		--build-arg BASE_IMAGE=ubuntu:20.04 .
+
+build_image_prognostic_run_gpu:
```

Comment:
Opted to build from an CUDA/ubuntu base image since it seemed easy, but this image is big (around 7-8 GB) w/ the CUDA libraries.